CMakeLists.txt

```diff
@@ -162,7 +162,16 @@ IF(BUILD_PYTHON_INTERFACE)
   MESSAGE(STATUS "The Python bindings of Pinocchio will be compiled along the main library. If you want to disable this feature, please set the option BUILD_PYTHON_INTERFACE to OFF.")
   FINDPYTHON(REQUIRED)
   SEARCH_FOR_BOOST_PYTHON(REQUIRED)
-  ADD_PROJECT_DEPENDENCY(eigenpy 2.6.2 REQUIRED)
+  ADD_PROJECT_DEPENDENCY(eigenpy 2.6.7 REQUIRED)
+
+  IF(BUILD_WITH_AUTODIFF_SUPPORT)
+    ADD_PROJECT_DEPENDENCY(pycppad 1.1.0 REQUIRED)
+    SET(CMAKE_CXX_STANDARD 11)
+    SET(CMAKE_CXX_STANDARD_REQUIRED ON)
```

Comment:
```suggestion
    CHECK_MINIMAL_CXX_STANDARD(11 REQUIRED)
```

Reply:
Otherwise this would lower the C++ standard on most recent compilers. eg. the main GCC on 18.04 defaults to C++14.

Reply:
In fact, we should set 11 at the beginning of the project.